src/js/callout.test.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-env jest */
+import { screen } from '@testing-library/dom';
+
+import initCallouts from './callout';
+
+const mockCallout = `<div class="cads-callout cads-callout-standard">
+  <h3>Callout title</h3>
+  <p>Example callout text</p>
+</div>`;
+
+test('should convert h3 in callout to a h2 if there is no existing h2', () => {
```

Comment:
I don't fully understand why this behaviour is in JS vs. passing in a `heading_level` to the component but good to have some tests for it.

Reply:
https://citizensadvice.atlassian.net/browse/NP-1096